src/utils.ts

```diff
@@ -42,6 +43,14 @@ export function consoleUrl(project: string, path: string): string {
   return `${api.consoleOrigin()}/project/${project}${path}`;
 }
 
+/**
+ * Create a Migrate to Firebase Console URL for the specified project.
+ */
+export function migrateToFirebaseLink(project: string): string {
```

Comment:
Do you have flexibility on this verb "migrate"?  It's not really a "migrate to Firebase" but more like "add Firebase" if that's possible?

Reply:
Definitely - I took 'migrate' from the query parameter in the console link, but if 'add Firebase' is more accurate lets use that

Reply:
ya, from the dev perspective, it's messaged as "add Firebase"

e.g., https://firebase.google.com/docs/reference/firebase-management/rest/v1beta1/projects/addFirebase